tools/oesign/oesign.c

```diff
@@ -671,3 +784,76 @@ int oedigest(const char* enclave, const char* conffile, const char* digest_file)
 done:
     return ret;
 }
+
+char hexchar2int(char ch)
+{
+    if (ch >= '0' && ch <= '9')
+        return ch - '0';
+    if (ch >= 'a' && ch <= 'f')
+        return 10 + ch - 'a';
+    if (ch >= 'A' && ch <= 'F')
+        return 10 + ch - 'A';
+    return 0;
+}
+
+ bool is_hex(char ch)
```

Comment:
You shouldn't need this function, just use the standard isxdigit() from ctype.h

Reply:
Done